src/test/java/com/znsio/rpap/BaseTest.java

```diff
@@ -2,47 +2,52 @@
 
 import com.znsio.applitools.integration.ApplitoolsInitializer;
 import com.znsio.reportportal.integration.properties.Config;
+import io.appium.java_client.AppiumDriver;
 import org.apache.log4j.Logger;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.WebDriverWait;
-
 import org.testng.annotations.AfterSuite;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
 import com.znsio.rpap.pages.Page;
 import com.znsio.rpap.utils.DriverFactory;
 
+import java.net.MalformedURLException;
 import java.time.Duration;
 import java.util.Properties;
 
 import static com.znsio.reportportal.integration.utils.ReportPortalLogger.logInfoMessage;
+import static com.znsio.rpap.utils.DriverFactory.WEB;
 
 public class BaseTest extends ApplitoolsInitializer {
-    protected static WebDriver webDriver;
+    protected static WebDriver driver;
     private static WebDriverWait wait;
     private static final Logger LOGGER = Logger.getLogger(BaseTest.class.getName());
-    private static final Properties config = Config.loadProperties(System.getProperty("CONFIG"));
     protected Page page;
 
     @BeforeSuite
-    public void suiteSetup() {
+    public void suiteSetup() throws MalformedURLException {
         log("Inside @BeforeSuite of " + BaseTest.class.getSimpleName());
-        webDriver = DriverFactory.launchWebApplication(webDriver, config.getProperty(Config.BROWSER),
-                config.getProperty("URL"));
-        wait = new WebDriverWait(webDriver, Duration.ofSeconds(Long.parseLong(config.getProperty("PAGE_LOAD_TIME"))));
-        ApplitoolsInitializer.driverSetupForApplitoolsInitializer(webDriver);
-        log("Browser is ready");
+        driver = DriverFactory.getDriver();
```

Comment:
why is the driver created in BeforeSuite? Now the tests cannot run in parallel

Reply:
Megha is working on it